exercises/practice/rtos-static-scheduling/.meta/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "blurb": "Given a task list, determine a schedule in which the tasks can execute sequentially with no overlap",
+  "authors": [
+    "CSwift8"
+  ],
+  "contributors": [
+    "Newby555"
```

Comment:
I don't need to be a contributor on this since you did your own config file. But if Grant helped you with the code at all then you should put him as an author or contributor (depending on how much he helped)